src/Microsoft.ML.TimeSeries/PolynomialModel/AbstractPolynomialModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// This class calculates f(x) = k0 + k1 * x + k2 * x^2 + k3 * x^3 ... kn * x^n with given x and coefs
+    /// </summary>
+    internal abstract class AbstractPolynomialModel
+    {
```

Comment:
Please combine all the `AbstractPolynomialModel` derived classes in this folder into the same file and remove the separate directory for PolynomialModel

Reply:
updated.

---
In reply to: [442525126](https://github.com/dotnet/machinelearning/pull/5202#discussion_r442525126) [](ancestors = 442525126)